src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerPushImage.groovy

```diff
@@ -56,7 +57,7 @@ class DockerPushImage extends AbstractDockerRemoteApiTask implements RegistryCre
 
         for (String image : images.get()) {
             AuthConfig authConfig = getRegistryAuthLocator().lookupAuthConfig(image, registryCredentials)
-            logger.quiet "Pushing image '${image}' to ${authConfig.registryAddress}."
+            logger.quiet "Pushing image '${image}' to ${getRegistryAuthLocator().getRegistry(image)}."
```

Comment:
This is missing the `get()` call on `image`.

Reply:
@Sineaggi Can you make the change? CI is failing right now. I am planning to release as soon as you make the change.

Reply:
I'm sure the code is right, the image is already a String. The test also passes while running in linux in a docker image.

Reply:
Looks like the CI issue was unrelated. Thanks for checking.